ansible/roles/resources/tasks/main.yml

```diff
@@ -297,6 +297,16 @@
          ((item.path|d() or item.dest|d() or item.name|d()) or item))
   tags: [ 'role::resources:paths' ]
 
+- name: Set custom file capabilities
+  capabilities:
+    path:       '{{ item.path  | d(item.name) }}'
+    capability: '{{ item.capability }}'
+    state:      '{{ item.state | d("present") }}'
+  with_flattened:
```

Comment:
While `with_flattened` is not (yet) deprecated, Ansible recommends using `loop` with the `flatten` filter: https://docs.ansible.com/ansible/latest/user_guide/playbooks_loops.html#with-flattened

Reply:
Thank you for pointing this out! I will prefer the `loop` `flatten` construct in the future.